electronics/details/details_flashing.md

```diff
@@ -10,14 +10,47 @@ After unpacking the LaunchPad,
 
 ## Step 2 - Install software to flash the firmware
 
-To be written.
+**TODO**:
 * How to install Code Composer
 * How to clone the required code
 
-## Step 3 - Flash the firmware
+## Step 3 - For development only: Run the Program via USB using CCS Debugger
+
+This is the default procedure during developement/debugging, as you can directly run updated versions of your code and the debugger allows full access to all global variables during runtime.
+
+From Code Composer:
+
+1. Compile your code, using the "Release" build.
+2. Make sure the board is powered and connected via USB to the PC. All three boot switche have to be in the upper position (see below).
+3. Start the Debugger by clicking on the bug icon in the tool bar: <img src="../images/ccs_button_debug.png"> 
```

Comment:
Markdown has actually support for images, I think it would be nicer to use this instead of injecting HTML: `![some title](../images/ccs_button_debug.png)`